### PR TITLE
Sergio bank ocr

### PR DESCRIPTION
**Pull request by Gioserden**

### CODE REVIEW

File: src/main/java/org/fundacionjala/coding/Sergio/BankORC.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+     * @param line3 IS THE THIRD LINE OF ENTRY
+     * @return THE NUMBER OF MAP IND
+     */
+    public static String convertEntryToNumber(String line1, String line2, String line3) {
+        String finalString = "";
+        //int sum = 3;
+        //int x = number.size();
+        for (int i = 0; i < MAXLENGTH; i = i + THREE) {
+           int j = i +  THREE;
+            finalString += number.get(line1.substring(i, j) + line2.substring(i, j) + line3.substring(i, j));
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/Sergio/BankORC.java]: Consider using a StringBuilder here

Comment (Gioserden) [reply src/main/java/org/fundacionjala/coding/Sergio/BankORC.java]: ok

**Comment by carledriss** [comment]
So far looks good
Try to complete it.

File: src/test/java/org/fundacionjala/coding/sergio/BankOCRTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+/**
+ * Created by Administrator on 5/16/2017.
+ */
+public class BankOCRTest {
+    /**
+     *
+     */
+    @Test
+    public void comprobateEntryNumbersOneToZero() {
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/sergio/BankOCRTest.java]: It is missing the scenario that we talked before

File: src/main/java/org/fundacionjala/coding/sergio/BankOCR.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+     * @param line1 IS THE FIRST LINE OF ENTRY.
+     * @param line2 IS THE SECOND LINE OF ENTRY.
+     * @param line3 IS THE THIRD LINE OF ENTRY.
+     * @return THE NUMBER OF MAP IND.
+     */
+    static String convertEntryToNumber(String line1, String line2, String line3) {
+        StringBuilder finalString = new StringBuilder();
+        for (int i = 0; i < MAXLENGTH; i = i + THREE) {
+            int j = i + THREE;
+            finalString.append(number.get(line1.substring(i, j) + line2.substring(i, j) + line3.substring(i, j)));
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/sergio/BankOCR.java]: Please don't concatenate with strings using +

File: src/main/java/org/fundacionjala/coding/sergio/BankOCR.java [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    private static final int SEVEN = 7;
+    private static final int EIGHT = 8;
+    private static final int MAXLENGTH = 27;
+    private static final int ZEROCONDITION = 0;
+    private static final int CHECKSUMCONDITION = 11;
+    private static final int ONE = 1;
+    private static final String S_ERR = "%s ERR";
+    private static final String S_ILL = "%s ILL";
+    private static final String NON_VALID_NUMBER = "?";
+    private static Map<String, Integer> number;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/sergio/BankOCR.java]: This attribute should be a constant.

**Comment by carledriss** [comment]
LGTM!

**Pull request merged**